content/en/docs/reference/glossary/Quantity.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Quantity
+id: Qty
+date: 2023-01-24
+full_link: https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/
+short_description: >
+ Quantity contains GB and GiB unit. 
+GiB: Gibibyte 
+GB: Gigabyte.
+
+aka: 
+tags:
+- user-type
```

Comment:
This is the wrong tag I'm afraid; a quantity _isn't_ a type of user.

Reply:
sorry, I just copied the format of other file. and forgot to change it. btw which type should I place.